docs.html

```diff
@@ -994,8 +998,127 @@ <h2 class="fs-m">JavaScript</h2>
+                    <div class="modal-container">
+                        <div class="modal">
+                            <div class="modal-header">
+                                <h1 class="fs-m">Modal Header</h1>
+                            </div>
+                            <p class="fs-s">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Repellat in odio impedit magni cum natus, numquam eaque sit culpa autem. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quasi quibusdam quae quis aliquid voluptatibus ad facilis voluptatem, quidem distinctio autem odit eligendi placeat voluptas quia, at fuga, explicabo aliquam nisi.</p>
+                            <div class="modal-cta">
+                                <button class="btn btn-secondary mr-sm">Save</button>
+                                <button class="btn btn-primary-outline">Close</button>
+                            </div>
+                        </div>
+                    </div>
+                </script>
+
+                <h2 class="fs-m">Live Demo of Modal</h2>
+                <button class="btn btn-secondary btn-show-modal">Click Here</button>
+
+                <h2 class="fs-m">Demo Snippet</h2>
+
+                <script type="text/plain" class="language-javascript line-numbers">
+                    <div class="modal-container-demo">
+                        <div class="modal modal-demo">
+                            <div class="modal-header">
+                                <h1 class="fs-m">Modal Header</h1>
+                            </div>
+                            <p class="fs-s">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Repellat in odio impedit magni cum natus, numquam eaque sit culpa autem. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quasi quibusdam quae quis aliquid voluptatibus ad facilis voluptatem, quidem distinctio autem odit eligendi placeat voluptas quia, at fuga, explicabo aliquam nisi.</p>
+                            <div class="modal-cta">
+                                <button class="btn btn-secondary mr-sm">Save</button>
+                                <button class="btn btn-primary-outline btn-close">Close</button>
+                            </div>
+                        </div>
+                    </div>
+
+                    <!-- ------- JavaScript -------  -->
+
+                    const modalContainer = document.querySelector(".modal-container-demo")
+                    const modalClose = document.querySelector(".btn-close");
+                    const showModal = document.querySelector(".btn-show-modal");
+
+                    showModal.addEventListener("click",() => {
+                        modalContainer.style.display = "block";
+                    })
+                    
+                    modalClose.addEventListener("click",()=>{
+                        console.log("clicked");
+                        modalContainer.style.display = "none";
+                    })
```

Comment:
I guess we are having same script code in index.js file 
do check it once

Reply:
This is for modal demo which is not included in components